librz/bin/format/mdmp/mdmp_pe.c

```diff
@@ -121,24 +116,10 @@ RzPVector /*<RzBinImport *>*/ *PE_(rz_bin_mdmp_pe_get_imports)(struct PE_(rz_bin
 		ptr->ordinal = imports[i].ordinal;
 		rz_pvector_push(ret, ptr);
 
-		if (!(rel = RZ_NEW0(RzBinReloc))) {
-			break;
-		}
-#ifdef RZ_BIN_PE64
-		rel->type = RZ_BIN_RELOC_64;
-#else
-		rel->type = RZ_BIN_RELOC_32;
-#endif
 		offset = imports[i].vaddr;
 		if (offset > pe_bin->vaddr) {
 			offset -= pe_bin->vaddr;
 		}
```

Comment:
this seems to be used only by the removed code. probably can be deleted too, including the variable `offset`